styleguide/_themes/derek/scss/patterns/quote.scss

```diff
@@ -1,3 +1,68 @@
+.quote {
+  background-color: $blue-gray;
+  padding: $half-padding;
+}
+
+.quote-container {
+  @include make-container;
```

Comment:
Are we ever placing these quotes in mini panels, because if so we shouldn't be defining a container here, and should probably just rely on standard bootstrap containers for that like we do with the other patterns.